contributorQualification.md

```diff
@@ -0,0 +1,62 @@
+# Aims
+
+Our most important aim with the trusted setup is to avoid all practical possibility of a single actor controlling the contributions. A secondary aim is to avoid censoring anyone who wishes to contribute. 
```

Comment:
```suggestion
Our most important aim with the trusted setup is to avoid all practical possibility of a single actor knowing all the secrets. A secondary aim is to avoid censoring anyone who wishes to contribute. 
```

IMO this is an important distinction as not knowing a secret is a very different safety assumption than not controlling all the contributors